Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -102,22 +110,33 @@ public struct OfflineMapAreasView: View {
         VStack(alignment: .center) {
             Text("No map areas")
                 .bold()
-            Text("You don't have any map areas yet.")
+            Text("There are no map areas defined for this web map.")
                 .font(.subheadline)
                 .foregroundStyle(.secondary)
+                .multilineTextAlignment(.center)
         }
         .frame(maxWidth: .infinity)
     }
     
     private var offlineDisabledView: some View {
```

Comment:
Please remove the explicit return statements and then annotate with `ViewBuilder`. Also, consolidate the strings.

```swift
@ViewBuilder private var offlineDisabledView: some View {
    let labelText = Text("Offline Disabled")
    let descriptionText = Text("Please ensure the web map is offline enabled.")
    if #available(iOS 17, *) {
        ContentUnavailableView {
            labelText
                .bold()
        } description: {
            descriptionText
        }
    } else {
        VStack(alignment: .center) {
            labelText
                .bold()
                .font(.title2)
            descriptionText
                .font(.subheadline)
                .foregroundStyle(.secondary)
        }
        .frame(maxWidth: .infinity)
    }
}
```